### ISSUE TITLE
Security vulnerability downstream

### ISSUE THREAD

**Issue by jdforsythe** [typestack/class-validator#208]
See https://github.com/typestack/routing-controllers/issues/408

class-validator@0.8.5 > validator@9.2.0

```sh
✗ Low severity vulnerability found on validator@9.2.0
- desc: Regular Expression Denial of Service (ReDoS)
- info: https://snyk.io/vuln/npm:validator:20180218
- from: @...@0.0.1 > routing-controllers@0.7.7 > class-validator@0.8.5 > validator@9.2.0
No direct dependency upgrade can address this issue.
```

**Comment by elthrasher** [comment]
Also reported here: https://github.com/typestack/class-validator/issues/202 and proposed fix here: https://github.com/typestack/class-validator/pull/206.

**Comment by NoNameProvided** [comment]
Fixed in 9122b4e and will be included in the next release.
